crtclient/docker/docker.go

```diff
@@ -28,6 +27,8 @@ import (
 	"github.com/contiv/netplugin/crtclient"
 	"github.com/samalba/dockerclient"
 	"github.com/vishvananda/netlink"
+
+	log "github.com/Sirupsen/logrus"
```

Comment:
I should have clarified: docker uses it, so we should be comfortable both with its stability and its compatibility. I was hoping we could use additional features; f.e. json logging would fit well with tools like fluentd and logstash.


> On May 21, 2015, at 3:22 PM, Vipin Jain <notifications@github.com> wrote:
> 
> In crtclient/docker/docker.go <https://github.com/contiv/netplugin/pull/76#discussion_r30854201>:
> 
> > @@ -28,6 +27,8 @@ import (
> >  	"github.com/contiv/netplugin/crtclient"
> >  	"github.com/samalba/dockerclient"
> >  	"github.com/vishvananda/netlink"
> > +
> > +	log "github.com/Sirupsen/logrus"
> @erikh <https://github.com/erikh> - given that logrus is compatible with standard logger, a simple replacement works. Should we use other features from it too (log with fields, hooks, etc.)? Perhaps we'd become incompatible quickly with other things then.
> 
> Another question: I checked the package is licensed under as "MIT License", which will work for us w.r.t. royalties etc for using this software. Do you know any other significant open-source effort(s) using this?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/contiv/netplugin/pull/76/files#r30854201>.
>